cypress.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "baseUrl": "http://localhost:3000"
```

Comment:
You have `localhost:3000` here but above, you set `CYPRESS_BASE_URL` to `localhost:5000`, are these not the same?

Reply:
They are not the same - I attempted to add this detail to the README: https://github.com/USSF-ORBIT/ussf-portal-client/blob/125f9fcbf9db298e6eeca69f681a8202e1ab3af2/README.md#static-site-vs-nextjs-server

tldr is `localhost:5000` is where the static site is served, and since that most closely represents what will be in production when we deploy next, that's what I have Cypress testing against. `localhost:3000` is the NextJS server & what is running during local development, which is why it's set as the default here.